static/js/rendered_markdown.js

```diff
@@ -235,6 +235,23 @@ export const update_elements = (content) => {
         });
     });
 
+    // We display the still image normally but show the GIF on hover.
+    content.find("a[href*='giphy.com/media']").each(function () {
```

Comment:
I guess this algorithm only works for GIPHY, not for other animated GIFs.  I'm not sure we have an alternative without a thumbnailing operation.

Reply:
Yeah, this will only work for GIPHY. I think we should experiment this on GIPHY GIFs and then extend it to other GIFs when we have proper support.